index.html

```diff
@@ -195,15 +221,15 @@ <h1 class="Section-title">
                     </p>
                     <form class="Contact-us__Form" id="fs-frm" name="simple-contact-form" accept-charset="utf-8" action="https://formspree.io/f/xdoybpyk" method="post">
                         <fieldset id="fs-frm-inputs">
-                          <input type="text" name="name" id="full-name" placeholder="First and Last" required="" maxlength="30">
-                          <input type="email" name="_replyto" id="email-address" placeholder="email@domain.tld" required="">
-                          <textarea rows="5" name="message" id="message" placeholder="Enter your message..." required="" maxlength="500"></textarea>
-                          <input type="hidden" name="_subject" id="email-subject" value="Contact Form Submission">
+                            <input type="text" name="name" id="full-name" placeholder="First and Last" required="" maxlength="30">
+                            <input type="email" name="_replyto" id="email-address" placeholder="email@domain.tld" required="">
+                            <textarea rows="5" name="message" id="message" placeholder="Enter your message..." required="" maxlength="500"></textarea>
+                            <input type="hidden" name="_subject" id="email-subject" value="Contact Form Submission">
                         </fieldset>
                         <div class="submit-container">
                             <input class="btn-primary" type="submit" value="Get in touch">
                         </div>
-                      </form>
+                    </form>
```

Comment:
- [ ] Please improve the look of your contact form by adding the appropriate margins
- [ ] Also add the missing social media links at the bottom of the page

**Currently, you have this:**

![amr-form-footer](https://user-images.githubusercontent.com/36542986/134591872-8da5d87e-41c4-4bf6-8bff-71d1f1d53ca3.jpg)

**The Figma template**

![amr-form-footer-orig](https://user-images.githubusercontent.com/36542986/134591990-fc20d11d-68df-4f96-b83b-12b59562dd86.jpg)